### DISCUSSION TITLE
Convert the drawable to an image - Not working

### DISCUSSION THREAD

**Discussion by FouadChaker** [dotnet/maui#5630] [Q&A]
Here I am working on a mobile application using maui. I should select 2 pictures from local space merge them, add some text and save the new image to local space.

- For the read process I used the file picker.
- For merging images I used maui graphics canvas.
- To save the result is the challenge. The documentation says the following :
"In code, the Drawable property of the GraphicsView object can then be accessed, and the ToImage method called:"
`IImage image = graphicsView.Drawable.ToImage(400, 500);`

The problem is that there is no method ToImage() in Drawable and no implementation is provided. `System.Drawing.Common` is windows only so I couldn't change the drawing api.

So, how should I approach converting the drawable to an image and saving it to local space?

**Comment by jsuarezruiz** [comment]
Could you open an enhancement in https://github.com/dotnet/Microsoft.Maui.Graphics/issues?
